docs/quickstart.rst

```diff
@@ -145,18 +145,25 @@ Note that this is not the same thing as::
   $ BORG_PASSPHRASE='complicated & long'; # missing "export"!
   $ ./yourborgjob.sh # will never see the above $BORG_PASSPHRASE value
 
+This is because ``export`` exposes variables to subprocesses, which borg may be
+one of. More on ``export`` can be found in the "ENVIRONMENT" section of the
+bash(1) man page, or the relevant POSIX_ spec.
+
+.. _POSIX: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_22.
+
 .. note:: ``sudo`` does not inherit your environment, so ``sudo borg`` will not
           have access to `BORG_PASSPHRASE` despite running the above example.
 
 To debug what your borg backup is truly seeing, for example if ``BORG_PASSPHRASE``
-is unexpectedly ignored, find its PID in another shell, E.g.::
+is unexpectedly ignored, find its PID and inspect its environment in another
+shell, E.g.::
 
-  $ BORG_PASSPHRASE # BAD: see explanation above
+  $ BORG_PASSPHRASE='complicated & long' # BAD: see explanation above
   $ ./yourborgjob.sh
   ...
   Enter passphrase for key ...:
 
-In another shell::
+Leave that ^ prompt hanging, and in another shell, inspect it's environment::
 
   $ ps aux | grep borg # find the python PID that borg is running as; E.g.: 123
   $ sudo cat /proc/123/environ  # inspect and verify
```

Comment:
cut those 10 lines down, see my previous comment. we must not blow up borg documentation too much.